bokehjs/src/lib/core/ui_events.ts

```diff
@@ -262,6 +266,11 @@ export class UIEvents implements EventListenerObject {
 
         const active_inspectors = this.toolbar.inspectors.filter(t => t.active)
         let cursor = "default"
+        let toolbar_is_visible = false
+
+        if (this._hit_test_canvas(e.sx, e.sy)) {
+          toolbar_is_visible = true
+        }
```

Comment:
Is there a reason for this kind of code instead of just `const toolbar_is_visible = this._hit_test_canvas(e.sx, e.sy)`, which is shorter and doesn't introduce a mutable variable?